akatsuki-co/_airbnb
# Stay search

Took a while but finally got search queries with dates.

I made the search queries strict, so if a city isn't an exact match it will not be added to the QuerySet.

To test:
go into Django admin and in the Stays console add a new booking (this is important, otherwise you just create a booking instance without it having been associated with a Stay).

Because I haven't started the CRUD functionality for Booking yet (this is my next step) this is the only way to check it right now.

I can show you tomorrow in more detail.
